test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -243,4 +298,83 @@ UCS_TEST_P(test_ucp_ep_reconfig, basic)
     r_receiver->verify_configuration(*r_sender);
 }
 
-UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_ep_reconfig, rc, "rc");
+/* TODO: Remove skip condition after next PRs are merged. */
+UCS_TEST_SKIP_COND_P(test_ucp_ep_reconfig, basic,
+                     !has_transport("rc_x") || !has_transport("rc_v"))
+{
+    run(EXCLUDE_IFACES);
+}
+
+UCS_TEST_P(test_ucp_ep_reconfig, request_reset, "PROTO_REQUEST_RESET=y")
+{
+    if (is_single_transport()) {
+        /* One side will consume all ifaces and the other side will have no ifaces left to use */
+        UCS_TEST_SKIP_R("exclude_iface requires at least 2 transports to work "
+                        "(for example DC + SHM)");
+    }
+
+    if (has_transport("ib") && !has_resource(sender(), "rc_verbs")) {
+        UCS_TEST_SKIP_R("No IB devs available, config will be non-wireup");
+    }
+
+    run(EXCLUDE_IFACES);
+}
+
+/* SHM causes lane reuse, disable for now. */
+UCS_TEST_SKIP_COND_P(test_ucp_ep_reconfig, resolve_remote_id,
+                     has_transport("shm") || is_self(), "RNDV_THRESH=0")
+{
+    /* There's no support for multiple paths + AM_ONLY in UCX (Remove in next
+     * PRs) */
+    modify_config("IB_NUM_PATHS", "1", SETENV_IF_NOT_EXIST);
+
+    if (has_transport("dc_x")) {
+        UCS_TEST_SKIP_R("extra path added for AM_LANE when using DC (will be"
+                        " removed)");
+    }
+
+    if (has_transport("tcp")) {
+        UCS_TEST_SKIP_R("lanes are reused (not supported yet)");
+    }
+
+    /* Create only AM_LANE to ensure we have only wireup EPs in
+     * configuration. */
+    run(EXCLUDE_IFACES, UCP_EP_INIT_CREATE_AM_LANE_ONLY, true);
+}
+
+UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_ep_reconfig, rc_x_v, "rc");
+UCP_INSTANTIATE_TEST_CASE(test_ucp_ep_reconfig);
+
+class test_reconfigure_asym_scale : public test_ucp_ep_reconfig {

```

Comment:
1. maybe move ASYMMETRIC_SCALE and related logic down to test_reconfigure_asym_scale class?
2. rename to test_reconfig_asymmetric